packages/cletus/src/helpers/render.tsx

```diff
@@ -90,6 +94,28 @@ export function renderOperation(
         <Text>{' → '}</Text>
         <Text color={op.error ? COLORS.ERROR_TEXT : undefined}>{summary}</Text>
       </Box>
+      
```

Comment:
For input and output let's do our own stringify. If   it's an array we us JSON stringify. If it's not an object we just String(x) it. If it is an object we output a bullet list using hyphens for each property. The property value is then JSON.stringified.